tests/functional/ext/check_elif/check_elif.py

```diff
@@ -25,3 +27,19 @@ def my_function():
                 if myint:
                     pass
                 myint = 4
+
+
+def _if_in_fstring_comprehension_with_elif():
+    order = {}
+    if "z" not in "false":
+        raise TypeError(
+            f" {', '.join(sorted(i for i in order or () if i not in vars))}"
```

Comment:
Am I mistaken or is `vars` not defined? And if so, this is probably a false negative. We might want to define it here so that we don't need to add the message here when we fix the false negative.

Reply:
``vars`` is a builtin but would need to be called on something (this is the code from psycopg https://github.com/psycopg/psycopg/blob/master/psycopg/psycopg/_queries.py#L159)

Reply:
Ah, yeah learn something new every day 😄

Reply:
29cf454f I renamed it to params because it's redefining a builtin. You're right that there should be an error in the previous code as vars is a function not an iterable and running the code would produce:
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: 'builtin_function_or_method' object is not iterable
```